tests/main.test.ts

```diff
@@ -77,6 +78,36 @@ describe("User tests", () => {
     });
   });
 
+  it("Should accurately calculate prices", () => {
+    const context = {
+      config: {
+        basePriceMultiplier: 3.0,
+      },
+    };
+    const testCases = [
```

Comment:
Probably should add more tests  with two decimals and more decimals.

Reply:
Maybe add a test with `0.73 hours` or things like that to force long decimals for the tests.

Reply:
Add a test with more decimals like gentlementlegen requested?